docs/os/modules/sysinitconfig/sysinitconfig.rst

```diff
@@ -352,6 +352,47 @@ use the flash area named ``FLASH_AREA_NFFS`` in the ``syscfg.yml`` file.
     syscfg.vals:
         NFFS_FLASH_AREA: FLASH_AREA_NFFS
 
+Conditional Settings
+~~~~~~~~~~~~~~~~~~~~
+
+Setings in most Mynewt YAML files can be made conditional on syscfg
+settings.  For example, a package might depend on a second package
+*only if a syscfg setting has a particular value*.  The condition can
+be the value of a single syscfg setting or an arbitrary expression
+involving many settings.
+
+Examples of Conditional Settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example 6
+`````````
+
+In this example, a package depends on `lib/pkg2` only if `MY_SETTING` has a
```

Comment:
Unlike Markdown where inline literals use a single backquote ("`"), reStructuredText uses double backquotes ("``") around the text.

Reply:
Thanks @utzig!  I will need to render this text locally to make sure I am not messing up more stuff.

By the way, I just noticed that there already was a "conditional settings" section at the bottom of the page!  I think the one I added is a bit better because it is more succinct and it talks about using arbitrary expressions.  So, I removed the old one.